templates/account/signup.html

```diff
@@ -6,16 +6,31 @@
   <div class="container-signinout">
     <img src="{% static 'images/bg-short.png' %}" alt="Background Image" class="d-none d-md-inline-flex">
     <div class="form-container">
-        <h2>Sign Up</h2>
-        <p>
-            Already have an account?
-            <a href="{% url 'account_login' %}">Sign in</a>
-        </p>
-        <form method="post" action="{% url 'account_signup' %}">
-        {% csrf_token %}
-        {{ form.as_p }}
```

Comment:
I'm concerned about the removal of the {{ form.as_p }} tag. 

Did you test registering as both For Personal and For Caretaker?

Reply:
Okey i can redo, no worries

Reply:
A redesign is great, as long as we ensure that all relevant fields are there and named and referenced accordingly. 

And before you do anything @JohnnySonTrinh - I just tested signing up on main, and for a caregiver works but it doesn't work for a Patient, so I'll look at that now.

Reply:
I found the issue with the signup, as far as I can tell it wasn't in your code.  I'm going to test your changes again with the issue fixed.